src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -135,6 +154,9 @@ public Object instanceMainPostParseArgs() {
 
         LoggingUtils.setLoggingLevel(VERBOSITY);  // propagate the VERBOSITY level to logging frameworks
 
+        // Log the configuration options:
+        ConfigUtils.logConfigFields(ConfigUtils.getOrCreate(GATKConfig.class));
```

Comment:
I would add this logging to the `printSettings()`, to allow API users to hide the logging...

Reply:
Good idea.  Fixed!